### DISCUSSION TITLE
about fantasy model

### DISCUSSION THREAD

**Discussion by LiuYang-1997** [meta-pytorch/botorch#963] [Q&A] [answered]
I am a novice, and when I see this term, I don’t understand what it means. Can someone explain me?Thank you very much.

**Comment by Balandat** [comment, marked as answer]
For look-ahead acquisition strategies, we are interested in reasoning about the possible observations we might make at a point in the domain, and how incorporating these would affect the model in the next iteration. A "fantasy observation" at a point `x_f` is nothing more than a sample `y_f = f(x_f)` drawn from the current model posterior at `x_f`. The "fantasy model" associated with that "fake"  observation is the current model conditioned on `(x_f, y_f)`.

**Reply by LiuYang-1997** [reply]
Thank you very much for solving the problem that has bothered me for several days.

**Comment by saitcakmak** [comment]
Fantasy (GP) model is the model obtained by conditioning the posterior on fantasy observations at given candidate points. A fantasy observation is an observation that is fantasized / drawn / simulated according to the model (predictive) posterior and likelihood at the given point. For the common Gaussian likelihood, this equivalent to drawing a realization from the posterior and adding the observation noise. They're commonly used in look-ahead type acquisition functions, such as knowledge gradient.

**Reply by LiuYang-1997** [reply]
Thank you very much for solving the problem that has bothered me for several days.
